Build fails on `nixpkgs-unstable`

Hello,

On `nixpkgs-unstable`, I get the following build error:

```
   Compiling anyhow v1.0.57
    Checking countme v2.0.4
    Checking text-size v1.1.0
    Checking rustc-hash v1.1.0
    Checking cfg-if v1.0.0
    Checking hashbrown v0.9.1
   Compiling directives v0.1.0 (/build/dummy-src/directives)
    Checking regex-syntax v0.6.25
error[E0514]: found crate `std` compiled by an incompatible version of rustc
  |
  = help: please recompile that crate using this compiler (rustc 1.61.0) (consider running `cargo clean` first)
  = note: the following crate versions were found:
          crate `std` compiled by rustc 1.60.0: /nix/store/ax84cgk6achib0sb90rdv6sacz9khw4k-rustc-1.60.0/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd_detect-ddd09637a51aa52e.rlib
          crate `std` compiled by rustc 1.60.0: /nix/store/ax84cgk6achib0sb90rdv6sacz9khw4k-rustc-1.60.0/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd-bb50407bcc8466c6.rlib
          crate `std` compiled by rustc 1.60.0: /nix/store/ax84cgk6achib0sb90rdv6sacz9khw4k-rustc-1.60.0/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd-bb50407bcc8466c6.so

error: requires `sized` lang_item

error: could not compile `directives` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
error: build failed
```

--- a/flake.nix
+++ b/flake.nix
@@ -2,6 +2,7 @@
   inputs = {
     flake-utils.url = "github:numtide/flake-utils";
     naersk.url = "github:nix-community/naersk";
+    naersk.inputs.nixpkgs.follows = "nixpkgs";
     nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
 
     flake-compat = {

